faqs/stocks/which-screener.md

```diff
@@ -9,75 +9,122 @@ description: >-
 
-You might see an opportunity or a red-flag from these screeners. However, it could easily just be a calculation error, or usage of incorrect formula, or not removing an outlier.
+The objective of this article is not to demean the websites but to show the importance of your own calculations.
 
-This article is to sensitize you that you might have to do more research rather than just going with the presented data from the UI of a screener.
+----
 
-## Examples of incorrect data in screeners
+## Examples of dividend yield
 
-We've come across some incorrect computations and results that are outright wrong. We've also informed the teams behind these screeners. At the time of writing this, these have not been addressed.  
-  
-It's our duty to warn others to be mindful of these, while they embark on their research.
+Lets consider the dividend yield of Britannia Industries as on 29th April 2021:
 
-Following are a few examples of such issues we'd noticed when we were doing this research into existing screeners.
+```
+Screener.in:    4.18%
+MoneyControl:   0.95%
+MorningStar.in: 1.78%
 
-### Gross Profit Margin
+Correct value:  ~2-5%
+Depends on your judgement on how repeatable these dividends are
+```
 
-The Gross Profit Margin on screener.in for all IT companies, including Tata Consultancy Services \(TCS\), is either 100%, or close to 100%.
+The differences are because of different treatments of interim dividends. While `screener.in` considers all the interim dividends of FY21, the `MoneyControl` considers only FY20's dividends. `MorningStar` considered only one of the two interim dividends (probably assuming the other one as a one time special dividend).
 
-![Gross Profit Margin of IT companies on screener.in - Dark Mode](../../.gitbook/assets/screener-gpm-dark.png)
 
-![Gross Profit Margin of IT companies on screener.in - Light Mode](../../.gitbook/assets/screener-gpm-light.png)
+Another interesting case is of Majesco. The dividend yield as on 29th April 2021 on various websites is:
 
-This basically implies that TCS has nearly ₹0 worth of expenses and all their sales is translated into gross profit.
+```
+Screener.in:              1,353 %
+MorningStar.in:           1,351 %
+ValueResearchOnline.com:  1,355 %
+(Yeah, 1000+% in all the cases)
 
-For those who’re unfamiliar, gross profit is the operational sales of a company minus its COGS \(**C**ost **O**f **G**oods **S**old\).
+Correct value: ~0%
+```
 
-Yeah, TCS doesn’t sell goods but it does have significant employee expenses. Since a services company like TCS doesn’t have goods in the traditional sense, we have to use their employee expenses to arrive at a meaningful gross profit margin figure.  
-  
-To say that gross profit margin of TCS is 100% is simply incorrect.
+The company sold off its entire business and distributed all of it as dividend. This is probably a one time dividend and the correct dividend yield should be around 0%.
```

Comment:
> The company sold off its entire business and distributed all of it as dividend. This is probably a one time dividend and the correct dividend yield should be around 0%

Can be

> This company in question, Majesco, sold off its entire business; and distributed all of liquid assets as dividend. This is probably a **one-time** dividend, and a sensible interpretation of dividend yield would be closer to 0%.

Reply:
Have corrected some details about the transaction. Have also rephrased the paragraph: https://github.com/indiainvestments/content/pull/49/commits/f2b659de40f2866b5206bd4e4b54438399f75946